docs/misc/prune-example.txt

```diff
@@ -79,24 +91,24 @@ List view
 Notes
 -----
 
-2015-12-31 is kept due to the --keep-daily 14 rule (because it is applied
-first), not due to the --keep-monthly or --keep-yearly rule.
+2015-12-31 is kept due to the `--keep-daily 14` rule (because it is applied
+first), not due to the `--keep-monthly` or `--keep-yearly` rule.
 
-The --keep-yearly 1 rule does not consider the December 31st backup because it
-has already been kept due to the daily rule. There are no backups available
-from previous years, so the --keep-yearly target of 1 backup is not satisfied.
-Because of this, the 2015-01-01 archive (the oldest archive available) is kept.
+The `--keep-yearly 1` rule does not consider the December 31st backup, because it
+has already been kept by the daily rule. 
+So far the --keep-yearly target of 1 backup is not satisfied.
+And because there are no backups available from previous years, prune keeps the 2015-01-01 archive (the oldest available).
 
-The --keep-monthly 6 rule keeps Nov, Oct, Sep, Aug, Jul and Jun. December is
-not considered for this rule, because that backup was already kept because of
-the daily rule.
+The `--keep-monthly 6` rule keeps Nov, Oct, Sep, Aug, Jul and Jun. December is not considered by this rule, because that backup is kept by the daily rule.
 
-2015-12-17 is kept to satisfy the --keep-daily 14 rule - because no backup was
-made on 2015-12-19. If a backup had been made on that day, it would not keep
-the one from 2015-12-17.
+2015-12-17 is kept to satisfy the `--keep-daily 14 rule`, because no backup was made on 2015-12-19. 
+If a backup had been made on that day, it would not keep the one from 2015-12-17.
 
-We did not include weekly, hourly, minutely or secondly rules to keep this
-example simple. They all work in basically the same way.
+We did not include weekly, hourly, minutely or secondly rules to keep this example simple. 
+They all work in basically the same way.
+Borg v1.2 onwards now "explains" which rule was used to keep a specific archive, with `--list` or `-v`.
 
-The weekly rule is easy to understand roughly, but hard to understand in all
-details. If interested, read "ISO 8601:2000 standard week-based year".
+The weekly rule is easy to understand roughly, but hard to understand in all details. 
+If you're interested, read "ISO 8601:2000 standard week-based year".
+If you are using prune in an international context, you should take into account the time zone in which prune is executed.
+Prune operates in the local time of the system running borg.
```

Comment:
The original document was word-wrapped to a reasonable width, as it is a .txt file.  Was it intended to change the line breaks to one per sentence?  I don't think that will look good.